### PR TITLE
Fix components description docs

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
All the components of ProductSummary were with the `ProductSummaryBuyButton` description. 

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductSummaryDescription/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-`ProductSummaryDescription` is a VTEX Component that render the buy button.
+`ProductSummaryDescription` is a VTEX Component that render the product description.
```
Comment (luciannojunior) [inline react/components/ProductSummaryDescription/README.md]:
```suggestion
`ProductSummaryDescription` is a VTEX Component that renders the product's description.
```

File: react/components/ProductSummaryImage/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-`ProductSummaryImage` is a VTEX Component that render the buy button.
+`ProductSummaryImage` is a VTEX Component that render the product image.
```
Comment (luciannojunior) [inline react/components/ProductSummaryImage/README.md]:
```suggestion
`ProductSummaryImage` is a VTEX Component that renders the product's image.
```

File: react/components/ProductSummaryName/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-`ProductSummaryName` is a VTEX Component that render the buy button.
+`ProductSummaryName` is a VTEX Component that render the product name.
```
Comment (luciannojunior) [inline react/components/ProductSummaryName/README.md]:
```suggestion
`ProductSummaryName` is a VTEX Component that renders the product's name.
```

**Comment by thayannevls** [comment]
@lbebber done, thanks!

**Pull request merged**